cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -974,7 +974,7 @@ class InheritanceConversionInstruction extends UnaryInstruction {
 
 /**
  * Represents an instruction that converts from the address of a derived class
- * to the address of a direct non-virtual base class.
+ * to the address of a direct base class.
```

Comment:
```suggestion
 * to the address of a base class.
```

Reply:
My typo (or cut-n-paste error) was worse that you thought. In the case of a virtual base class, it's not necessarily even a direct base class.

Reply:
Thanks, turns out this observation is potentially relevant in https://github.com/Semmle/ql/pull/2638.